layouts/partials/archived/extend-footer.html

```diff
@@ -1,7 +1,8 @@
 
 <div class="flex items-center justify-between text-sm text-neutral-500 dark:text-neutral-400 py-6">
     <p>
-        The EVERSE is funded by the European<br>Commission <a href="https://ec.europa.eu/info/funding-tenders/opportunities/portal/screen/opportunities/topic-details/horizon-infra-2023-eosc-01-02">HORIZON-INFRA-2023-EOSC-01-02</a> call
+        This project has received funding from the European Union’s Horizon<br>
```

Comment:
I am not sure about the use of `<br>` here. It only helps the layout in a very specific way and if you resize the window it can break the line in a weird and not so nice way. I would take that out.

Reply:
Ok, I followed the way the current sentence was written but will do